server/model/pipeline.go

```diff
@@ -20,50 +20,50 @@ import (
 )
 
 type Pipeline struct {
-	ID                  int64                  `json:"id"                      xorm:"pk autoincr 'pipeline_id'"`
-	RepoID              int64                  `json:"-"                       xorm:"UNIQUE(s) INDEX 'pipeline_repo_id'"`
-	Number              int64                  `json:"number"                  xorm:"UNIQUE(s) 'pipeline_number'"`
-	Author              string                 `json:"author"                  xorm:"INDEX 'pipeline_author'"`
-	Parent              int64                  `json:"parent"                  xorm:"pipeline_parent"`
-	Event               WebhookEvent           `json:"event"                   xorm:"pipeline_event"`
-	Status              StatusValue            `json:"status"                  xorm:"INDEX 'pipeline_status'"`
-	Errors              []*types.PipelineError `json:"errors"                  xorm:"json 'pipeline_errors'"`
-	Created             int64                  `json:"created_at"              xorm:"pipeline_created"`
+	ID                  int64                  `json:"id"                      xorm:"pk autoincr 'id'"`
+	RepoID              int64                  `json:"-"                       xorm:"UNIQUE(s) INDEX 'repo_id'"`
+	Number              int64                  `json:"number"                  xorm:"UNIQUE(s) 'number'"`
+	Author              string                 `json:"author"                  xorm:"INDEX 'author'"`
+	Parent              int64                  `json:"parent"                  xorm:"parent"`
+	Event               WebhookEvent           `json:"event"                   xorm:"event"`
+	Status              StatusValue            `json:"status"                  xorm:"INDEX 'status'"`
+	Errors              []*types.PipelineError `json:"errors"                  xorm:"json 'errors'"`
+	Created             int64                  `json:"created_at"              xorm:"created NOT NULL DEFAULT 0 'created'"`
```

Comment:
```suggestion
	Created             int64                  `json:"created_at"              xorm:"created_at NOT NULL DEFAULT 0 'created_at'"`
```

Would use `_at` suffix for all the timestamps to be consistent with json.

Why is created_at twice in it? Same for `updated_at`. Does it enable some xorm timestamp updating routine?

Reply:
I would just rename the xorm columns here?

Reply:
Looks like we have evenly used `created` and `created_at` feel free to pick your preference for xorm and we will migrate the json property in 3.0